### PR TITLE
docs(nx-dev): put all angular recipes together PENDING

### PR DESCRIPTION
**Pull request by meeroslav**
## Current Behavior
Recipes related to Angular are scattered in `Adopting Nx`, `Adopting Nx From Angular CLI` and `Other`

## Expected Behavior
All Angular related recipes should be in the same `Angular` section:
- Migrations from CLI and JS
- NgRx specific
- Tailwind with Angular

Compare the menu structure at: https://nx-dev-git-fork-meeroslav-fix-docs-move-angular-recipes-nrwl.vercel.app/recipes

## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/shared/reference/sitemap.md [hunk 1]
```diff
@@ -145,14 +145,18 @@
...
-    - [From AngularJS](/recipes/adopting-nx/migration-angularjs)
     - [Preserving Git Histories](/recipes/adopting-nx/preserving-git-histories)
     - [Manual migration](/recipes/adopting-nx/manual)
-  - [Adopting Nx From Angular CLI](/recipes/adopting-nx-angular)
-    - [From Angular CLI](/recipes/adopting-nx-angular/migration-angular)
-    - [From Angular CLI to Integrated](/recipes/adopting-nx-angular/angular-integrated)
-    - [From Angular CLI Manually](/recipes/adopting-nx-angular/angular-manual)
-    - [From Multiple Angular CLI Repos](/recipes/adopting-nx-angular/angular-multiple)
+  - [Angular](/recipes/adopting-nx-angular)
+    - [Migrating Angular CLI](/recipes/adopting-nx-angular/migration-angular)
```
Comment (Coly010) [inline docs/shared/reference/sitemap.md:151]:
```suggestion
    - [Migrating from the Angular CLI](/recipes/adopting-nx-angular/migration-angular)
```

File: docs/shared/reference/sitemap.md [hunk 2]
```diff
@@ -145,14 +145,18 @@
...
+    - [Migrating Angular CLI to Integrated](/recipes/adopting-nx-angular/angular-integrated)
```
Comment (Coly010) [inline docs/shared/reference/sitemap.md:152]:
```suggestion
    - [Migrating from the Angular CLI to Integrated Monorepo](/recipes/adopting-nx-angular/angular-integrated)
```

File: docs/shared/reference/sitemap.md [hunk 3]
```diff
@@ -145,14 +145,18 @@
...
+    - [Migrating Angular Manually](/recipes/adopting-nx-angular/angular-manual)
```
Comment (Coly010) [inline docs/shared/reference/sitemap.md:153]:
```suggestion
    - [Migrating from the Angular CLI Manually](/recipes/adopting-nx-angular/angular-manual)
```

**Review by Coly010 — COMMENTED** [review]
I think this looks ok, but I think the menu item text should be a bit more clear.

There are more areas affected than the ones I've indicated here, but they should all be changed the same

**Review by Coly010 — APPROVED** [review]

**Comment by juristr** [comment]
@meeroslav should this be funneled into the broader discussion on how to reorg some of the stuff we have on the docs page? Like, if we add a section for Angular, we should probably have one for React, Next, Node,...

Right the side-menu organization is kept agnostic from tech but from an Nx focus like migrating etc.. which I agree has its flaws, which is why we should discuss this in general.

As we had already discussed, there are different angles

- Nx specific recipes
- Tech specific recipes related on how to use Nx with <tech-stack> but not necessarily related to any Nx tech specific plugin
- Nx plugin specific recipes

**Comment by meeroslav** [comment]
We already had segment called `Adopting Nx form Angular CLI` so that's not new.

I believe an Angular user might come to recipes looking for Angular-related content or a framework agnostic one.
There shouldn't be any recipes in the `Other` as each recipe needs to belong somewhere.

Grouping NgRx-related with Angular migration recipes makes sense, in my opinion.

Same should go for React specific recipes.

But we can have a discussion on the topic first.

File: docs/map.json [hunk 4]
```diff
@@ -1043,29 +1038,54 @@
           ]
         },
         {
-          "name": "Adopting Nx From Angular CLI",
+          "name": "Angular",
           "id": "adopting-nx-angular",
-          "description": "Advanced guides to adopt Nx coming from the Angular CLI",
+          "description": "Advanced guides to using Nx with Angular ecosystem",
```
Comment (isaacplmann) [inline docs/map.json:1043]:
```suggestion
          "description": "Advanced guides to using Nx within the Angular ecosystem",
```

File: docs/map.json [hunk 5]
```diff
@@ -1043,29 +1038,54 @@
...
           "itemList": [
             {
-              "name": "From Angular CLI",
+              "name": "Migrating Angular CLI",
```
Comment (isaacplmann) [inline docs/map.json:1046]:
```suggestion
              "name": "Migrating from the Angular CLI",
```

File: docs/map.json [hunk 6]
```diff
@@ -1043,29 +1038,54 @@
...
               "id": "migration-angular",
               "file": "shared/migration/migration-angular"
             },
             {
-              "name": "From Angular CLI to Integrated",
+              "name": "Migrating Angular CLI to Integrated",
```
Comment (isaacplmann) [inline docs/map.json:1051]:
```suggestion
              "name": "Migrating from the Angular CLI to an Integrated Repo",
```

File: docs/map.json [hunk 7]
```diff
@@ -1043,29 +1038,54 @@
...
               "file": "shared/migration/angular-manual"
             },
             {
-              "name": "From Multiple Angular CLI Repos",
+              "name": "Migrate Multiple Angular Repos",
               "id": "angular-multiple",
               "file": "shared/migration/angular-multiple"
+            },
+            {
+              "name": "Migrate AngularJS",
```
Comment (isaacplmann) [inline docs/map.json:1066]:
```suggestion
              "name": "Migrate an AngularJS Repo",
```

**Review by isaacplmann — CHANGES_REQUESTED** [review]
Similar comments to @Coly010 but the changes should be made in map.json rather than sitemap.md.

**Comment by meeroslav** [comment]
Closing as we will refactor the docs further

**Pull request closed without merging**